nltk/app/collocations_app.py

```diff
@@ -13,12 +13,35 @@
 
 from six.moves import queue as q
 from six.moves.tkinter_font import Font
-from six.moves.tkinter import (Button, END, Frame, IntVar, LEFT, Label, Menu,
-                               OptionMenu, SUNKEN, Scrollbar, StringVar,
-                               Text, Tk)
-
-from nltk.corpus import (cess_cat, brown, nps_chat, treebank, sinica_treebank, alpino,
-                         indian, floresta, mac_morpho, machado, cess_esp)
+from six.moves.tkinter import (
+    Button,
+    END,
+    Frame,
+    IntVar,
+    LEFT,
+    Label,
+    Menu,
+    OptionMenu,
+    SUNKEN,
+    Scrollbar,
+    StringVar,
+    Text,
+    Tk,
+)
```

Comment:
I'm not sure whether it's okay but I don't necessary agree that listing one submodule per line here is useful.

Reply:
The whole point of [https://github.com/ambv/black](url) is to avoid having conversations about preferred formatting style. From their readme:

_Black is the uncompromising Python code formatter. By using it, you agree to cede control over minutiae of hand-formatting. In return, Black gives you speed, determinism, and freedom from pycodestyle nagging about formatting. You will save time and mental energy for more important matters._

More discussion on [reddit](https://www.reddit.com/r/Python/comments/8lmnzr/thoughts_on_black/).

If you prefer, whole files (`--exclude`) or blocks of text can be excluded from formatting with `# fmt: off` and `# fmt: on`